src/main/java/core/basesyntax/state/FigureSupplier.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.state;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+
+public class FigureSupplier {
+    private static String [] figureArray = {"Circle", "Isosceles Trapezoid", "Rectangle",
+            "Right Triangle", "Square"};
+
+    public static Figure getRandomFigure() {
+        int randomNumber = (int)(Math.random() * figureArray.length);
+
+        if (randomNumber == 0) {
```

Comment:
You can use switch case construction for this

Reply:
done